mod/reportedcontent/start.php

```diff
@@ -79,22 +63,48 @@ function reportedcontent_page_handler($page) {
  * Add report user link to hover menu
  */
 function reportedcontent_user_hover_menu($hook, $type, $return, $params) {
```

Comment:
nope.. can't do. Need to hack it in via the view extension. This hook is called in an ajax loaded view